Updated  Contributing.md (#1038)
keptn/keptn.github.io: CONTRIBUTING.md

```
hugo
```
## Contributing guide  
1. While contributing, make sure you use a separate branch than the Main branch 
2. If you think you can make some patch to fix the issue, fork the repo, and then make a pull
 request.

> 3. Its always a good practice to squash your commits before creating a final pull reuest, avoid conflicting commits. 

Reviewer: ```suggestion
3. It's always a good practice to squash your commits before creating a final pull request, avoid conflicting commits. 
```

> 4. If you dont think you can contribute back to the patch/bug, open a new issue

Reviewer: ```suggestion
4. If you don't think you can contribute back to the patch/bug, open a new issue
```

> 5. While rasing the issue try to be as detailed(but also specific) as possible to allow others to understand the bug.

Reviewer: ```suggestion
5. When submitting the issue try to be as detailed(but also specific) as possible to allow others to understand the bug.
```
